src/uproot/streamers.py

```diff
@@ -553,9 +553,10 @@ def read_members(self, chunk, cursor, context, file):
 
         self._members["fTypeName"] = _canonical_typename(cursor.string(chunk, context))
 
-        if self._members["fType"] == 11 and self._members["fTypeName"] in (
-            "Bool_t" or "bool"
-        ):
+        if self._members["fType"] == 11 and self._members["fTypeName"] in {
+            "Bool_t",
+            "bool",
+        }:
```

Comment:
This is also now a fix.

Reply:
Yikes! I was curious about how it ended up that way, since I thought it might have originally been

```python
(
    self._members["fTypeName"] == "Bool_t"
    or self._members["fTypeName"] == "bool"
)
```

incorrectly converted to use `in`. But no: I just wrote it that way from the beginning.

https://github.com/scikit-hep/uproot5/blob/76f6511976b7e1ed702218fd10b5b0b7e0e1164d/src/uproot/streamers.py#L556-L558